package.json

```diff
@@ -158,5 +158,6 @@
         "react"
       ]
     }
-  ]
+  ],
+  "packageManager": "yarn@1.22.21+sha256.dbed5b7e10c552ba0e1a545c948d5473bc6c5a28ce22a8fd27e493e3e5eb6370"
```

Comment:
What's this about?

Reply:
gahhh yarn keeps adding that

Reply:
we're supposed to specify what version of yarn the repo should use through `packageManager`

i never got the chance to set this up properly, so it keeps trying to add it whenever i run `yarn install`

maybe i should just set it up 🤔 hold on

Reply:
hmm looks a bit complicated, something for another day...

Reply:
Yeah no clue 🫡